### PR TITLE
Add UTC timestamp output to STDOUT log message.

### PR DESCRIPTION
**Pull request by bsodmike**
I wanted to see some timestamp details in the output log; this is a tiny change though.  Included the `chrono` crate as it has some helpful features for manipulating timezones etc.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -11,6 +12,10 @@ mod tests;
 
 use check::{Check, Status};
 
+fn get_utc_time() -> chrono::DateTime<chrono::Utc> {
```
Comment (bsodmike) [inline src/main.rs]: We might like to move this to a separate module, especially if we want to add further function calls to mutate the output timezone etc. Thoughts?

**Comment by bsodmike** [comment]
@ckauhaus quick heads up :)

File: src/main.rs (hunk 1 above)
Comment (bsodmike) [reply src/main.rs]: This update has been made.

**Comment by ckauhaus** [comment]
Sorry for the late reply. I won't accept this PR since it breaks the output format. check_journal should adhere to the Nagios plugin guidelines. Although these are quite dates, they still present a common basis for several monitoring systems like Nagios, check_mk, sensu and others.

**Pull request closed without merging**